examples/scripts/README.rst

```diff
@@ -155,6 +160,21 @@ Sparsity
       Video Decomposition via Robust PCA
 
 
+Machine Learning
+^^^^^^^^^^^^^^^^
+
+   `ct_astra_modl_train_foam2.py <ct_astra_modl_train_foam2.py>`_
+      CT Reconstruction with MoDL
+   `ct_astra_unet_train_foam2.py <ct_astra_unet_train_foam2.py>`_
+      CT with UNet for Denoising of FBP
+   `deconv_modl_train_foam.py <deconv_modl_train_foam.py>`_
```

Comment:
Some index entries are broken now that `foam` became `foam1`.

Reply:
Fixed.